packages/@aws-cdk/aws-glue/lib/job.ts

```diff
@@ -711,7 +711,7 @@ export class Job extends JobBase {
     if (config.extraJars && config.extraJars?.length > 0) {
       args['--extra-jars'] = config.extraJars.map(code => this.codeS3ObjectUrl(code)).join(',');
     }
-    if (config.extraPythonFiles && config.extraPythonFiles.length > 0) {
+    if (JobLanguage.PYTHON === config.language && config.extraPythonFiles && config.extraPythonFiles.length > 0) {
```

Comment:
I don't think it should ignore them silently if the language is not Python

Reply:
good call!

Reply:
Proper handling in https://github.com/aws/aws-cdk/pull/12506/commits/094929cccc6f576044dbc0fedfc04789b1265bee